modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -517,4 +547,18 @@ export default class LargeVideoManager {
     onLocalFlipXChange(val) {
         this.videoContainer.setLocalFlipX(val);
     }
+
+    /**
+     * Dispatches an action to update the known resolution state of the
+     * large video.
+     *
+     * @private
+     * @returns {void}
+     */
+    _updateVideoResolutionStatus() {
+        const { height } = this.videoContainer.getStreamSize();
+        const isCurrentlyHD = height >= config.minHDHeight;
```

Comment:
Since you are here, can you fix this so portrait orientation is taken into account? A mobile device could have a resolution of 480x640, which would be considered HD because 640 because it's > 540.